tools/schema-explorer/static/css/style.css

```
@@ -0,0 +1,2021 @@
+  :where(.schema-spa-host, .schema-embed) .schema-var-table th,
+  :where(.schema-spa-host, .schema-embed) .schema-var-table td {
+    display: block;
+    width: 100% !important;
+  }
+
+  :where(.schema-spa-host, .schema-embed) .schema-var-table thead {
+    position: absolute;
+    width: 1px !important;
+    height: 1px;
+    overflow: hidden;
+    clip: rect(0 0 0 0);
```

Stylelint (17.13.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)